doc/man3/EVP_mdc2.pod

```diff
@@ -0,0 +1,53 @@
+=pod
+
+=head1 NAME
+
+EVP_mdc2
+- MDC-2 For EVP
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ const EVP_MD *EVP_mdc2(void);
+
+=head1 DESCRIPTION
+
+MDC-2 (Modification Detection Code 2 or Meyer-Schilling) is a cryptographic
+hash function based on a block cipher.
+
+=over 4
+
+=item EVP_mdc2()
+
+The MDC-2DES algorithm of using MDC-2 with the DES block cipher. It produces a
+128-bit output from a given input.
+
+=back
+
+=head1 RETURN VALUES
+
+These functions return a B<EVP_MD> structure that contains the
+implementation of the symmetric cipher. See L<EVP_MD_meth_new(3)> for
+details of the B<EVP_MD> structure.
+
+=head1 CONFORMING TO
+
+N/A.
```

Comment:
Here's another one.

I heard your argument about having this as a placeholder, in case we find a reference later on...  Unfortunately, placeholders aren't in our habits.  As a matter of fact, we have had some, but removed them all at some point.

I did check to see if I could quickly find something to put here.  Alas...

Reply:
Feel free :-)

Reply:
I'm still trying to find out which exact algorithm it is in that document... one sec 😄

Reply:
Resolved.